instrumentation/github.com/gin-gonic/gin/otelgin/gin.go

```diff
@@ -17,7 +17,7 @@ import (
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/metric"
 	"go.opentelemetry.io/otel/propagation"
-	oteltrace "go.opentelemetry.io/otel/trace"
+	"go.opentelemetry.io/otel/trace"
```

Comment:
This change is unrelated to this PR. Could you split it into another one?

Reply:
Yes, I have reverted this change